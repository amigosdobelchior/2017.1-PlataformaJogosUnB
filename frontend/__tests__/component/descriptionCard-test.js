import React from 'react';
import {Card} from "semantic-ui-react";
import renderer from 'react-test-renderer';
var expect = required('expect');

test('Test render DescriptionCard', () => {
  const component = renderer.create(
      <Card fluid>
        <Card.Content header='Descrição' description="content"/>
        <Card fluid>
            <Card.Content header='Prêmios' description="content"/>
        </Card>
      </Card>
	);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
