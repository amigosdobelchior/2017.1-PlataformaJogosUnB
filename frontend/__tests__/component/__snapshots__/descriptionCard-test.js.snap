// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render DescriptionCard 1`] = `
<div
  className="ui fluid card"
  href={undefined}
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      Descrição
    </div>
    <div
      className="description"
    >
      content
    </div>
  </div>
  <div
    className="ui fluid card"
    href={undefined}
    onClick={[Function]}
  >
    <div
      className="content"
    >
      <div
        className="header"
      >
        Prêmios
      </div>
      <div
        className="description"
      >
        content
      </div>
    </div>
  </div>
</div>
`;
